.rubocop.yml

```diff
@@ -46,22 +46,16 @@ Style/SymbolArray:
 Style/HashSyntax:
   EnforcedStyle: ruby19_no_mixed_keys
 
-# This cop no longer recognized.
-# has_key? and has_value? are far more readable than key? and value?
-#Style/DeprecatedHashMethods:
-#  Enabled: false
-
-# Options: format, sprintf, percent
-#Style/FormatString:
-#  EnforcedStyle: false
+# Frozen string literal: http://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/FrozenStringLiteralComment
+Style/FrozenStringLiteralComment:
+  Enabled: false
```

Comment:
Do you understand what this does? It's still not clear to me. Something to do with Ruby 3:

- https://stackoverflow.com/a/38326572/6763239